spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe UsersController, type: :controller do
```

Comment:
Torn on this, part of me feels we can just delete the whole thing as the scope just tests AR default behaviour and scope is tested elsewhere. I like having this sort of thing more as documentation around a requirement. That requirement word makes me agree on feature, but it's mainly the exception I care about. 

TLDR: feature it is :)